src/session_p.h

```diff
@@ -735,6 +746,16 @@ struct nc_server_opts {
     } cert_exp_notif;
 #endif /* NC_ENABLED_SSH_TLS */
 
+    /**
+     * @brief Entry for a UNIX socket path configured via API (Hidden).
+     *
+     * These paths are not part of the YANG configuration and are managed manually by the application.
+     */
+    struct nc_server_unix_path_entry {
+        char *endpt_name;       /**< Name of the endpoint using this path. */
+        char *path;             /**< The actual filesystem path for the socket. */
+    } *unix_env_paths;  /**< Array of UNIX socket paths set via API (sized-array, see libyang docs). */
```

Comment:
`env` is misleading, please use something else, they are not environment variables. Just `unix_paths` may be fine or even `unix_api_paths`, up to you what you choose.